2018-06-08T00:10:27.487Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '8787' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T00:10:27.488Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '8787' in collection with ID 'Asset:org.ayuda.Loan' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '8787' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n    at channel.queryByChaincode.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-06-08T00:17:01.353Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '9' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T00:17:01.354Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '9' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '9' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T00:38:14.759Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '987' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T00:38:14.760Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '987' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '987' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T00:38:54.363Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '987' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T00:38:54.365Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '987' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '987' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T00:39:55.672Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '6e03ea0b1f7bcfa5dcde84628d4a6d3c12bcfb61e2a1c5e43a835316f7b294be' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T00:39:55.672Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '6e03ea0b1f7bcfa5dcde84628d4a6d3c12bcfb61e2a1c5e43a835316f7b294be' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '6e03ea0b1f7bcfa5dcde84628d4a6d3c12bcfb61e2a1c5e43a835316f7b294be' in collection with ID 'Participant:org.ayuda.Borrower' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T00:42:23.747Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '9999' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T00:42:23.747Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '9999' in collection with ID 'Asset:org.ayuda.Loan' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '9999' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n    at channel.queryByChaincode.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-06-08T00:42:38.721Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '9999' in collection with ID 'Transaction:org.ayuda.CreateLoan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T00:42:38.722Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '9999' in collection with ID 'Transaction:org.ayuda.CreateLoan' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '9999' in collection with ID 'Transaction:org.ayuda.CreateLoan' does not exist)\n    at channel.queryByChaincode.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-06-08T01:08:15.010Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: You did not enter the amount to lend)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T01:08:15.010Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: You did not enter the amount to lend)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: You did not enter the amount to lend)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T01:08:57.295Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: You did not enter the amount to lend)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T01:08:57.295Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: You did not enter the amount to lend)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: You did not enter the amount to lend)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T02:57:14.468Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T02:57:14.468Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:10:26.107Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:10:26.107Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:11:59.303Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:11:59.304Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:17:33.014Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:17:33.014Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:19:28.817Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:19:28.817Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:19:34.961Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:19:34.962Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '120' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:21:36.255Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:21:36.256Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:23:48.359Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:23:48.360Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:24:43.236Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:24:43.236Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:27:48.815Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:27:48.816Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:28:15.801Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:28:15.802Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:29:26.906Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:29:26.906Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:29:41.969Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:29:41.970Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:30:11.657Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:30:11.658Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:31:16.447Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:31:16.447Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:31:54.653Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:31:54.653Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '44' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-06-08T03:40:55.895Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '1314' in collection with ID 'Asset:org.ayuda.Loan' does not exist)
    at /home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-06-08T03:40:55.895Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '1314' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Object with ID '1314' in collection with ID 'Asset:org.ayuda.Loan' does not exist)\n\n    at _initializeChannel.then.then.then.then.catch (/home/souradeep/.nvm/versions/node/v8.9.3/lib/node_modules/composer-rest-server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
